tools/update_runnablejobs.py

```diff
@@ -243,6 +226,49 @@ def add_jobs_to_jobpriority(new_data=None, priority=1, timeout=0, set_expired=Fa
                           .values(buildsystem=_buildsystem)
             conn.execute(statement)
 
+
+def add_jobs_to_jobpriority(new_data=None, priority=1, timeout=0, set_expired=False):
```

Comment:
you do not use priority or timeout here, they should be proxied to the next function